var/spack/repos/builtin/packages/ramulator/ramulator_sha_7d2e723_libPatch.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/Makefile b/Makefile
+index ea340c8..314113d 100644
+--- a/Makefile
++++ b/Makefile
+@@ -9,7 +9,7 @@ OBJS := $(patsubst $(SRCDIR)/%.cpp, $(OBJDIR)/%.o, $(SRCS))
+ #   g++ 4.x due to an internal compiler error when processing lambda functions.
+ CXX := clang++
+ # CXX := g++-5
+-CXXFLAGS := -O3 -std=c++11 -g -Wall
++CXXFLAGS := -O3 -std=c++11 -g -Wall -fpic

```

Comment:
This will only work for certain compilers, it would be better to query `self.compiler.cxx_pic_flag` and use that. You should have access to `self` in a `patch(self)` method.